cmd/kops/create_cluster.go

```diff
@@ -535,8 +574,22 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 	if len(nodes) == 0 {
 		g := &api.InstanceGroup{}
 		g.Spec.Role = api.InstanceGroupRoleNode
-
 		g.ObjectMeta.Name = "nodes"
+
+		subnetNames := sets.NewString()
```

Comment:
We explicitly initialize it - it is safer, and we don't necessarily have all the information later in the GCE case